src/commonTest/kotlin/com/charleskorn/kaml/testobjects/TestObjects.kt

```diff
@@ -96,3 +97,13 @@ data class TestClassWithNestedTaggedNode(
     val text: String,
     val node: YamlTaggedNode,
 )
+
+@Serializable
+@JvmInline
+value class TestInline<out T>(val value: T)
```

Comment:
This seems to be testing two things at once: inline classes and generics. Could you please remove the generics so the intent is clearer and there's no confusion?